extension/llm/export/quantizer_lib.py

```diff
@@ -193,3 +199,34 @@ def get_qnn_quantizer(
     ), "Currently qnn backend only supports QnnQuantizer via pt2e flow"
     qnn_quantizer.add_custom_quant_annotations(custom_annotations)
     return qnn_quantizer, quant_dtype
+
+
+def get_coreml_quantizer(pt2e_quantize: str):
+    if pt2e_quantize == "coreml":
+        config = LinearQuantizerConfig.from_dict(
+            {
+                "global_config": {
+                    "quantization_scheme": QuantizationScheme.affine,
+                    "activation_dtype": torch.quint8,
+                    "weight_dtype": torch.qint8,
+                    "weight_per_channel": True,
+                }
+            }
+        )
+        quantizer = CoreMLQuantizer(config)
+
+    elif pt2e_quantize == "coreml_qc4":
+        raise NotImplementedError("4-bit coreml quantizer is still under development")
+
+    elif pt2e_quantize == "coreml_xnnpack":
+        config = get_symmetric_quantization_config(is_per_channel=True, is_dynamic=False)
```

Comment:
Is it for per channel quantization? If so, let's create a todo here to add per channel in coreml quantizer?

Reply:
ditto